code/js/modules/BaseController.js

```diff
@@ -210,11 +216,23 @@
    * @return {{song: {String}, artist: {String}, isPlaying: {Boolean}, siteName: {String}}}
    */
   BaseController.prototype.getStateData = function() {
+    var vCanDislike = !!(this.selectors.dislike && this.doc().querySelector(this.selectors.dislike)),
```

Comment:
Not a fan of this naming scheme. Please change to `_canDislike`, `_canPlayPrev`, etc...